client/src/components/Invites.js

```diff
@@ -30,11 +35,81 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const testInvites = [];
-
+const testInvitesIn = [
+  {
+    avatar: tempAvatar,
+    email: 'friend0@aol.com',
+    name: 'friend0',
+    status: 'pending',
+  },
+  {
+    avatar: tempAvatar,
+    email: 'friend1@aol.com',
+    name: 'friend1',
+    status: 'pending',
+  },
+  {
+    avatar: tempAvatar,
+    email: 'friend2@aol.com',
+    name: 'friend2',
+    status: 'pending',
+  },
+  {
+    avatar: tempAvatar,
+    email: 'friend3@aol.com',
+    name: 'friend3',
+    status: 'pending',
+  },
+  {
+    avatar: tempAvatar,
+    email: 'friend4@aol.com',
+    name: 'friend4',
+    status: 'pending',
+  },
+  {
+    avatar: tempAvatar,
+    email: 'friend5@aol.com',
+    name: 'friend5',
+    status: 'pending',
+  },
+  {
+    avatar: tempAvatar,
+    email: 'friend6@aol.com',
+    name: 'friend6',
+    status: 'pending',
+  },
+  {
+    avatar: tempAvatar,
+    email: 'friend7@aol.com',
+    name: 'friend7',
+    status: 'pending',
+  },
+  {
+    avatar: tempAvatar,
+    email: 'friend8@aol.com',
+    name: 'friend8',
+    status: 'pending',
+  },
+  {
+    avatar: tempAvatar,
+    email: 'friend9@aol.com',
+    name: 'friend9',
+    status: 'pending',
```

Comment:
We want our dummy data to match our model - which only has status, referrer, toEmail, and createdAt.